XenAdmin/Wizards/ImportWizard/ImageVMConfigPage.cs

```diff
@@ -47,9 +47,9 @@ internal partial class ImageVMConfigPage : XenTabPage
 		public ImageVMConfigPage()
 		{
 			InitializeComponent();
-            m_upDownMemory.Minimum = m_upDownMemory.Increment = VM.DEFAULT_MEM_MIN_IMG_IMPORT;
-            m_upDownMemory.Maximum = VM.DEFAULT_MEM_ALLOWED;
-            m_ctrlError.HideError();
+            m_upDownMemory.Minimum = m_upDownMemory.Increment = (decimal)VM.DEFAULT_MEM_MIN_IMG_IMPORT / Util.BINARY_MEGA;
+            m_upDownMemory.Maximum = (decimal)VM.DEFAULT_MEM_ALLOWED / Util.BINARY_MEGA;

```

Comment:
I assume the value given to the `m_upDownMemory` control is already in Megabytes and this is to correct the Min and Max to be in those same units.

Reply:
you're exactly right

best to do it here than to refactor `m_upDownMemory` to use bytes :)

Reply:
When the page is populated the value is set to the minimum. which is in  MB. Then when the user changes it, the value is passed down as is and added to the OVF envelope with unit MB. See https://github.com/xenserver/xenadmin/blob/7c873334399a63f4293497b10c4d89ddda736ebb/XenAdmin/Wizards/ImportWizard/ImportWizard.cs#L272 and https://github.com/xenserver/xenadmin/blob/7c873334399a63f4293497b10c4d89ddda736ebb/XenOvfApi/OVF.cs#L1410